app/views/chargeback_rate/copy.html.haml

```diff
@@ -0,0 +1 @@
+= render :partial => 'edit'
```

Comment:
Are we doing AJAX partial reloads in the controller? If no, we can just move the partial here and get rid of it.

Reply:
> Are we doing AJAX partial reloads in the controller? If no, we can just move the partial here and get rid of it.

I need this template to be there for direct links/buttons to work, this is rendering a shared partial that's being used by add/copy/edit buttons.

Reply:
I know that, my suggestion is to move the contents of `_edit` to here.